backend/src/payment/index.ts

```diff
@@ -70,20 +63,29 @@ class Payment {
     return this.provider.create(order, args);
   }
 
-  verifySuccess(req: express.Request): Promise<ISuccessResponse> {
+  verifySuccess(req: express.Request): Promise<void> {
     return this.provider.verifySuccess(req);
   }
 
-  verifyCancel(req: express.Request): Promise<ICancelResponse> {
+  verifyCancel(req: express.Request): Promise<void> {
     return this.provider.verifyCancel(req);
   }
 
-  checkStatus(payment_id: string, payment_url: string): Promise<IStatusResponse> {
-    return this.provider.checkStatus(payment_id, payment_url);
+  checkStatus(order_id: number, payment_id: string, payment_url: string): Promise<IStatusResponse> {
+    return this.provider.checkStatus(order_id, payment_id, payment_url);
   }
 }
 
 // factory for creating payment providers
 export default function(provider: string): Payment {
   return new Payment(provider);
+}
+
+//helper functions for payment provider implementations
+export function getOrderId(order_id: number): string {
```

Comment:
"orderId" doesn't sound like correct name here, as it conflicts with the order_id that's given as arg. Dunno what would be better though.

Reply:
I changed it to orderIdToName, which is at least little bit better.